v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+---
+title: Verify Constraint Conformance
+summary: Verify that your cluster's data replication, data placement, and zone configurations are working as expected.
+keywords: availability zone, zone config, zone configs, zone configuration, constraint, constraints, 
+toc: true
+---
+
+<span class="version-tag">New in v19.2:</span> Several new and updated tables (listed below) are available to help you verify your cluster's constraint conformance. For example, you can check the following types of constraints:
+
+- See what data is under-replicated or unavailable
+- Show localities that are at risk in the event of node failures
+- See if any of your cluster's data placement constraints are being violated, and whether the cluster is recovering from those violations.
+
+{{site.data.alerts.callout_info}}
+The information on this page assumes you are familiar with [replication zones](configure-replication-zones.html) and [partitioning](partitioning.html).
+{{site.data.alerts.end}}
+
+## Conformance reporting tables
+
+The following new and updated tables are available for verifying constraint conformance.
+
+- [`system.replication_stats`](#system-replication_stats) shows information about whether data is under-replicated, over-replicated, or unavailable.
+- [`system.replication_constraint_stats`](#system-replication_constraint_stats) shows a list of violations to any data placement requirements you've configured, along with information about whether the cluster is actively recovering from those violations.
+- [`system.replication_critical_localities`](#system-replication_critical_localities) shows which localities in your cluster have data that is at risk in the event of node failures.
+- [`system.reports_meta`](#system-reports_meta) lists the IDs and times when the reports ran that generated the data in the `system.replication_*` tables.
+- [`crdb_internal.zones`](#crdb_internal-zones) can be used with the tables above to figure out the database and table names where the non-conforming or at-risk data is located.
```

Comment:
Hm, I don't understand.  From my user POV, I don't really know what "zone IDs" mean unless I can get at the table names etc. where the data "is located" (from my POV) that I can access from SQL.  I only introduced the use of `crdb_internal.zones` because it allowed me to map from zone IDs to databases and table names.

Are you saying any or all of:

1. the support for this mapping is not correct?
2. we should not try to do the mapping in this doc?  (i.e., remove use of `crdb_internal.zones`?)
3. something completely else (?)